src/selectors/exportContext.ts

```diff
@@ -69,8 +76,11 @@ export const exportContext = (state: State, context: Context, format: MimeType =
     ? `${childrenPrefix}\n${childrenFiltered.map(exportChild).join('\n')}${childrenPostfix}${format === 'text/html' ? indent === 0 ? tab0 : tab1 : ''}`
     : ''
 
-  const text = `${tab0}${linePrefix}${head(context)}${exportedChildren && format === 'text/html' ? tab1 : ''}${exportedChildren}${linePostfix}`
+  let text = `${tab0}${linePrefix}${head(context)}${exportedChildren && format === 'text/html' ? tab1 : ''}${exportedChildren}${linePostfix}`
 
+  if (format === 'text/plain') {
+    text = stripHTMLTag(text)
+  }
```

Comment:
It would be better to use two separate variables instead of overriding `text`. We try to avoid mutations as much as possible.

Reply:
I see you approved and merged. I will update this in my next PR.

Reply:
Yeah, if they are minor points sometimes I'll approve and merge and just mention it for future reference. You don't have to fix it this round.